tests/test_meta/test_zero_inflated_regressor.py

```diff
@@ -0,0 +1,24 @@
+"""Test the LADRegressor."""
+
+import pytest
+
+from sklego.common import flatten
+from sklego.meta import ZeroInflatedRegressor
+from sklego.testing import check_shape_remains_same_regressor
+from tests.conftest import general_checks, select_tests, regressor_checks
+
+
+@pytest.mark.parametrize("test_fn", [check_shape_remains_same_regressor])
+def test_lad(test_fn):
+    regr = ZeroInflatedRegressor()
+    test_fn(ZeroInflatedRegressor.__name__, regr)
+
```

Comment:
I might add a test with an "obvious example" to demonstrate that it indeed is able to separate the zeros from a normal regression pattern.

Reply:
I used the one from the feature requests, is this ok?